src/main/scala/com/normation/rundeck/plugin/resources/rudder/DataModel.scala

```diff
@@ -102,21 +101,19 @@ final case class NodeId(value: String)
 final case class GroupId(value: String)
 
 final case class Group(
-    id     : GroupId
-  , name   : String
-  , nodeIds: Set[NodeId]
-  , enable : Boolean
-  , dynamic: Boolean
+    id: GroupId,
+    name: String,
+    nodeIds: Set[NodeId],
+    enable: Boolean,
+    dynamic: Boolean
 )
 
 //////////////////////////////// Error container ////////////////////////////////
 
-/**
- * All the method that can fail are of the Failable[T] type (defined in
- * package.scala, alias to Either[ErrorMsg,T]).
- * ErrorMsg is just a container for an Error, with an human readable
- * message, and optionnaly the root exception which caused it.
- */
+/** All the method that can fail are of the Failable[T] type (defined in
```

Comment:
Can we keep the other format (first line without text, only one space) ? This part is not linked to scala3 strictly, and it's better to keep consistency across projects for things that are not part of the experiment.